lib/core/network/removeErrorStack.js

```diff
@@ -31,12 +33,20 @@ const util = require('util');
  * @returns {*} return the data minus the stack trace
  */
 module.exports = data => {
-  if (process.env.NODE_ENV !== 'development') {
-    if (util.isError(data)) {
-      delete data.stack;
+  if (util.isError(data)) {
+    if (global.NODE_ENV !== 'development') {
+      data.stack = undefined;
+    }
+    else {
+      data.stack = data.stack.split('\n').map(hilightUserCode).join('\n');
+    }
+  }
+  else if (data && data.content && data.content.error) {
+    if (global.NODE_ENV !== 'development') {
+      data.content.error.stack = undefined;
     }
-    else if (data && data.content && data.content.error) {
-      delete data.content.error.stack;
+    else {
+      data.content.error.stack = data.content.error.stack.split('\n').map(hilightUserCode).join('\n');
```

Comment:
```js
      data.content.error.stack = data.content.error.stack.split('\n').map(hilightUserCode).join('\n');
```
Maybe have a function to do this would be better instead of repeating this bit of code everywhere ?

Reply:
I can't because I use this method in a custom way in the `kerror/index.js` file